src/databricks/labs/ucx/workspace_access/generic.py

```diff
@@ -299,9 +299,9 @@ def _crawl(self) -> list[WorkspaceObjectInfo]:
                 continue
             raw = obj.as_dict()
             yield WorkspaceObjectInfo(
```

Comment:
And where are the changes to this dataclass?

Reply:
I'm not sure what changes are necessary.